Formula/jenv.rb

```diff
@@ -15,12 +15,24 @@ def install
   end
 
   def caveats
-    <<~EOS
-      To activate jenv, add the following to your #{shell_profile}:
+    if preferred == :fish
+      s = <<~EOS
+        To activate jenv, run the following commands:
 
-        export PATH="$HOME/.jenv/bin:$PATH"
-        eval "$(jenv init -)"
-    EOS
+      EOS
+      s << "  fish_add_path #{bin}\n" if bin.directory?
+      s << "  echo 'status --is-interactive; and source (jenv init -|psub)' >> #{shell_profile}\n"
+      s << "  cp #{libexec}/fish/jenv.fish ~/.config/fish/functions/jenv.fish\n"
+      s << "  cp #{libexec}/fish/export.fish ~/.config/fish/functions/export.fish\n\n"
```

Comment:
Why append all of these to `s` instead of just including them when `s` is first defined?

Reply:
removed

Reply:
I didn't realize I didn't need them. Not very familiar with Ruby :)